### PR TITLE
replaced mixin word with fieldgroup

### PR DESCRIPTION
**Pull request by prabhum2**
Please link to the issue #…

### CODE REVIEW

File: components/fieldgroups/campaign-member/campaign-member-components.schema.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/campaign-member"],
-  "description": "This mixin is used to capture B2B campaign member specific relationship attributes that are represented as it is in source and also used for merge/unmerge rules analysis if and when needed.",
+  "description": "This field group is used to capture B2B campaign member specific relationship attributes that are represented as it is in source and also used for merge/unmerge rules analysis if and when needed.",
```
Comment (hanessian) [inline components/fieldgroups/campaign-member/campaign-member-components.schema.json:15]: "Capture B2B campaign member specific relationship attributes that are represented as it is in source and also us..."

Make the description as short as possible so users can see the purpose of the field group quickly

File: components/fieldgroups/campaign/campaign-components.schema.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/campaign"],
-  "description": "This mixin is used to capture b2b campaign specific relationship attributes that are represented as it is in source and also used for merge/unmerge rules analysis if and when needed.",
+  "description": "This field group is used to capture b2b campaign specific relationship attributes that are represented as it is in source and also used for merge/unmerge rules analysis if and when needed.",
```
Comment (hanessian) [inline components/fieldgroups/campaign/campaign-components.schema.json:15]: "Captures B2B campaign..."

Let's capitalize B2B wherever used to stay consistent.

File: components/fieldgroups/campaign/campaign-details.schema.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/campaign"],
-  "description": "This mixin is used to capture properties related to business campaign details.",
+  "description": "This field group is used to capture properties related to business campaign details.",
```
Comment (hanessian) [inline components/fieldgroups/campaign/campaign-details.schema.json:19]: Captures properties..

File: components/fieldgroups/experience-event/experienceevent-application.schema.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
         "xdm:application": {
           "title": "Application",
           "$ref": "https://ns.adobe.com/xdm/context/application",
-          "description": "This mixin is used to capture application information related to an ExperienceEvent, including the name of the application, app version, installs, launches, crashes, and closures. It could be either the application targeted by the event like the send of a push notification or the application originating the event such as a click, or a login."
+          "description": "This field group is used to capture application information related to an ExperienceEvent, including the name of the application, app version, installs, launches, crashes, and closures. It could be either the application targeted by the event like the send of a push notification or the application originating the event such as a click, or a login."
```
Comment (hanessian) [inline components/fieldgroups/experience-event/experienceevent-application.schema.json:22]: "Captures application information, including the name of the..

Comment (hanessian) [reply components/fieldgroups/experience-event/experienceevent-application.schema.json:22]: note the removal of "related to an EE"

File: components/fieldgroups/opportunity/opportunity-details.schema.json [hunk 5]
```diff
@@ -15,7 +15,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/opportunity"],
-  "description": "This mixin is used to capture and maintain additional opportunity attributes that are used by industry crm systems.",
+  "description": "This field group is used to capture and maintain additional opportunity attributes that are used by industry crm systems.",
```
Comment (hanessian) [inline components/fieldgroups/opportunity/opportunity-details.schema.json:18]: Captures and maintains additional...

File: components/fieldgroups/product/product-catalog-category.schema.json [hunk 6]
```diff
@@ -13,7 +13,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/product"],
-  "description": "This mixin is used to capture properties related to product catalog category.",
+  "description": "This field group is used to capture properties related to product catalog category.",
```
Comment (hanessian) [inline components/fieldgroups/product/product-catalog-category.schema.json:16]: Captures properties...

File: components/fieldgroups/product/product-catalog.schema.json [hunk 7]
```diff
@@ -13,7 +13,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/product"],
-  "description": "This mixin is used to capture properties related to product catalog.",
+  "description": "This field group is used to capture properties related to product catalog.",
```
Comment (hanessian) [inline components/fieldgroups/product/product-catalog.schema.json:16]: Captures properties...

File: components/fieldgroups/product/product-category.schema.json [hunk 8]
```diff
@@ -13,7 +13,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/product"],
-  "description": "This mixin is used to capture properties related to product category.",
+  "description": "This field group is used to capture properties related to product category.",
```
Comment (hanessian) [inline components/fieldgroups/product/product-category.schema.json:16]: Captures properties..

File: components/fieldgroups/product/product-identifiers.schema.json [hunk 9]
```diff
@@ -13,7 +13,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/product"],
-  "description": "This mixin is used to capture properties related to product identifiers generic to all industries.",
+  "description": "This field group is used to capture properties related to product identifiers generic to all industries.",
```
Comment (hanessian) [inline components/fieldgroups/product/product-identifiers.schema.json:16]: Captures properties..

File: components/fieldgroups/product/product-measurement.schema.json [hunk 10]
```diff
@@ -13,7 +13,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/product"],
-  "description": "This mixin is used to capture properties related to product measurements.",
+  "description": "This field group is used to capture properties related to product measurements.",
```
Comment (hanessian) [inline components/fieldgroups/product/product-measurement.schema.json:16]: Captures properties..

File: components/fieldgroups/profile/b2b-person-details.schema.json [hunk 11]
```diff
@@ -15,7 +15,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
-  "description": "This mixin is used to capture B2B Person specific attributes. A B2B Person could be a contact/lead etc.",
+  "description": "This field group is used to capture B2B Person specific attributes. A B2B Person could be a contact/lead etc.",
```
Comment (hanessian) [inline components/fieldgroups/profile/b2b-person-details.schema.json:18]: Captures B2B..

File: components/fieldgroups/profile/profile-inferred-person.schema.json [hunk 12]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
-  "description": "This mixin is used to capture inferred details about a person where they are not directly avaialble.",
+  "description": "This field group is used to capture inferred details about a person where they are not directly avaialble.",
```
Comment (hanessian) [inline components/fieldgroups/profile/profile-inferred-person.schema.json:15]: Captures inferred...

File: components/fieldgroups/profile/profile-other-work-details.schema.json [hunk 13]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
-  "description": "This mixin is used to capture information regarding additional/other work-related details of the individual such as work address, work email, work phone number, and organizations to which the profile belongs.",
+  "description": "This field group is used to capture information regarding additional/other work-related details of the individual such as work address, work email, work phone number, and organizations to which the profile belongs.",
```
Comment (hanessian) [inline components/fieldgroups/profile/profile-other-work-details.schema.json:15]: Captures information..

File: extensions/adobe/b2b/bizible/bizible-opportunity-details.schema.json [hunk 14]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/opportunity"],
-  "description": "This mixin is used to capture and maintain additional Bizible opportunity attributes. They extend the ones used by industry crm systems.",
+  "description": "This field group is used to capture and maintain additional Bizible opportunity attributes. They extend the ones used by industry crm systems.",
```
Comment (hanessian) [inline extensions/adobe/b2b/bizible/bizible-opportunity-details.schema.json:15]: Captures and..

File: extensions/adobe/b2b/bizible/bizible-person-details.schema.json [hunk 15]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
-  "description": "This mixin is used to capture Bizible B2B Person specific attributes. A B2B Person could be a contact/lead etc.",
+  "description": "This field group is used to capture Bizible B2B Person specific attributes. A B2B Person could be a contact/lead etc.",
```
Comment (hanessian) [inline extensions/adobe/b2b/bizible/bizible-person-details.schema.json:15]: Captures...

File: extensions/adobe/b2b/marketo/marketo-web-url.schema.json [hunk 16]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
-  "description": "Use this mixin for web url which does not confine with URI format.",
+  "description": "Use this field group for web url which does not confine with URI format.",
```
Comment (hanessian) [inline extensions/adobe/b2b/marketo/marketo-web-url.schema.json:15]: What does this field group capture? It's not clear from the description. 

"Captures web url, which need not be in URI format"

?

File: extensions/adobe/experience/consumer-experienceevent.schema.json [hunk 17]
```diff
@@ -9,7 +9,7 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Consumer ExperienceEvent  Mixin",
   "type": "object",
-  "description": "This mixin groups a set of standard fields to describe the behavior of an individual. This mixin can be used to express behavior of a consumer related to digital content consumption (web, mobile app), online or off-line purchases. The use of this standard represention allows for a single representation for data producers and consumers of consumer behavior in Experience Platform",
+  "description": "This field group is a set of standard fields to describe the behavior of an individual. This field group can be used to express behavior of a consumer related to digital content consumption (web, mobile app), online or off-line purchases. The use of this standard represention allows for a single representation for data producers and consumers of consumer behavior in Experience Platform",
```
Comment (hanessian) [inline extensions/adobe/experience/consumer-experienceevent.schema.json:12]: Looks like the title of this needs to be changed...to consumer experienceEvent Field Group

Description could be shortened to " A set of ...

File: extensions/adobe/experience/intelligentServices/profile-journeyai-engagementscores.schema.json [hunk 18]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
-  "description": "This mixin contains JourneyAI Engagement Scores, which provides individual customer's propensities to engage with a message.",
+  "description": "This field group contains JourneyAI Engagement Scores, which provides individual customer's propensities to engage with a message.",
```
Comment (hanessian) [inline extensions/adobe/experience/intelligentServices/profile-journeyai-engagementscores.schema.json:15]: Contains..

File: extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json [hunk 19]
```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
-  "description": "This mixin contains JourneyAI Send Time Optimization scores, which provides the time preferences for message deliveries.",
+  "description": "This field group contains JourneyAI Send Time Optimization scores, which provides the time preferences for message deliveries.",
```
Comment (hanessian) [inline extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json:15]: Contains..

**Review by hanessian — COMMENTED** [review]
Please see comments on each item. Overall feedback is to not include "this field group contains.." as that take up most of the UI description space, without telling the user anything. better is.."Contains information..." etc. Let me know if you have any questions! Thanks :)

File: components/fieldgroups/opportunity/opportunity-details.schema.json [hunk 20]
```diff
@@ -15,7 +15,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/opportunity"],
-  "description": "This mixin is used to capture and maintain additional opportunity attributes that are used by industry crm systems.",
+  "description": "Captures and maintain additional opportunity attributes that are used by industry crm systems.",
```
Comment (hanessian) [inline components/fieldgroups/opportunity/opportunity-details.schema.json:18]: and maintains

**Review by hanessian — APPROVED** [review]

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**